vrp-cli/src/lib.rs

```diff
@@ -185,6 +185,7 @@ mod c_interop {
         matrices: *const *const c_char,
         matrices_len: usize,
         config: *const c_char,
+        geojson: bool,
```

Comment:
This change requires adjusting documentation and examples here:

https://github.com/reinterpretcat/vrp/tree/master/docs/src/examples/interop
https://github.com/reinterpretcat/vrp/tree/master/examples/jvm-interop

Reply:
https://github.com/reinterpretcat/vrp/pull/108/commits/c8bd7e87770442031962e7c1b20145bc8dea2352 - thx!